prow/plugins/releasenote/releasenote.go

```diff
@@ -148,24 +149,30 @@ func handleComment(gc githubClient, log *logrus.Entry, ic github.IssueCommentEve
 			return err
 		}
 	}
+
+	labels := sets.String{}
+	for _, label := range ic.Issue.Labels {
+		labels.Insert(label.Name)
+	}
 	// Remove all other release-note-* labels if necessary.
 	return removeOtherLabels(
 		func(l string) error {
 			return gc.RemoveLabel(org, repo, number, l)
 		},
 		releaseNoteNone,
 		allRNLabels,
-		ic.Issue.Labels,
+		&labels,
 	)
 }
 
-func removeOtherLabels(remover func(string) error, label string, labelSet []string, currentLabels []github.Label) error {
+func removeOtherLabels(remover func(string) error, label string, labelSet []string, currentLabels *sets.String) error {
```

Comment:
`currentLabels` doesn't need to be a pointer. `sets.String` is already a reference type.